new/PRS14020Ptolemy.xml

```diff
@@ -49,10 +49,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <persName xml:lang="en" xml:id="n1">Ptolemy XII Auletes</persName>
                     <persName xml:lang="gez" xml:id="n2">በጥሊሞስ፡ </persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n2">Baṭlimos</persName>
-                    <persName xml:lang="grc" xml:id="n3">Ptolemaios Neos Dionysos Philopatōr Philadelphos</persName>
-                    <persName xml:lang="grc" type="normalized" corresp="#n2">Πτολεμαῖος Νέος Διόνυσος Φιλοπάτωρ Φιλάδελφος</persName>
-                    <persName xml:lang="grc" xml:id="n4">Ptolemaios Auletes</persName>
-                    <persName xml:lang="grc" type="normalized" corresp="#n2">Πτολεμαῖος Αὐλητής</persName>
+                    <persName xml:lang="grc" xml:id="n3">Πτολεμαῖος Νέος Διόνυσος Φιλοπάτωρ Φιλάδελφος</persName>
+                    <persName xml:lang="grc" type="normalized" corresp="#n2">Ptolemaios Neos Dionysos Philopatōr Philadelphos</persName>
+                    <persName xml:lang="grc" xml:id="n4">Πτολεμαῖος Αὐλητής</persName>
+                    <persName xml:lang="grc" type="normalized" corresp="#n2">Ptolemaios Auletes</persName>
```

Comment:
it should be `corresp="#n4"`, new PR is needed

Reply:
Done.